tests/conftest.py

```diff
@@ -8,7 +8,7 @@
 from selenium.webdriver.chrome.service import Service
 from webdriver_manager.chrome import ChromeDriverManager
 
-multiprocessing.set_start_method("fork")  # Required on macOSX
+# multiprocessing.set_start_method("fork")  # Required on macOSX
```

Comment:
Rather than commenting this out can we make it conditional as per: https://github.com/communitiesuk/funding-service-design-notification/blob/703a2eb907d4012671521d8ad47a306ea01f6f77/tests/conftest.py#L13-L14